conda/core/package_cache.py

```diff
@@ -411,7 +411,12 @@ def make_actions_for_dist(dist, record):
                 target_pkgs_dir=pc_entry_writable_cache.pkgs_dir,
                 target_extracted_dirname=pc_entry_writable_cache.dist.dist_name,
             )
-            return None, extract_axn
+            dr_axn = DumpRecordAction(
+                target_pkgs_dir=pc_entry_writable_cache.pkgs_dir,
+                target_extracted_dirname=pc_entry_writable_cache.dist.dist_name,
+                record=record,
```

Comment:
I'm not sure what you mean by that comment. Are you saying some of the record fields may not be correct ?